### PR TITLE
[RFR] Backport master 1.3.5

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: packages/react-admin/src/mui/field/ReferenceField.js [hunk 1]
```diff
@@ -2,13 +2,22 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import get from 'lodash.get';
+import { withStyles } from 'material-ui/styles';
+import compose from 'recompose/compose';
+import classenames from 'classnames';
```
Comment (fzaninotto) [inline packages/react-admin/src/mui/field/ReferenceField.js]: typo

File: packages/react-admin/src/mui/field/ReferenceField.js [hunk 2]
```diff
@@ -135,6 +145,7 @@ ReferenceField.propTypes = {
     allowEmpty: PropTypes.bool.isRequired,
     basePath: PropTypes.string.isRequired,
     children: PropTypes.element.isRequired,
+    classes: PropTypes.object,
```
Comment (fzaninotto) [inline packages/react-admin/src/mui/field/ReferenceField.js]: add to default props, too (`{}`) to ease tests.

File: packages/react-admin/src/mui/field/ReferenceField.spec.js [hunk 3]
```diff
@@ -16,6 +20,7 @@ describe('<ReferenceField />', () => {
                 reference="bar"
                 basePath=""
                 crudGetManyAccumulate={crudGetManyAccumulate}
+                classes={classes}
```
Comment (fzaninotto) [inline packages/react-admin/src/mui/field/ReferenceField.spec.js]: should not be necessary once you have default prop for `classes`.

File: packages/react-admin/src/mui/field/ReferenceField.js (hunk 2 above)
Comment (djhi) [reply packages/react-admin/src/mui/field/ReferenceField.js]: Done

**Pull request merged**